src/client/src/reducers/index.ts

```diff
@@ -21,5 +23,33 @@ const rootReducer = combineReducers({
   dependencyInfo
 });
 
-export type AppState = ReturnType<typeof rootReducer>;
+export type AppState = ReturnType<typeof appReducer>;
+
+const rootReducer = (state: AppState | undefined, action: RootAction) => {
+  let passedState: any;
+
+  if (action.type === WIZARD_INFO_TYPEKEYS.RESET_WIZARD) {
```

Comment:
Would it make sense to have the `ResetReducer` as a separate reducer file?

Reply:
Hmm the problem with that is that it will not apply to the entire root 😦. The point of this is to have the handling applied to the entire root state, if that makes sense? I may not be understanding your proposition though, if you think so can you give me an example of how you'd want the code to be?

Reply:
I think I get your point. ResetWizard wants to take state from different places and set it to some value but it is not a child of the root reducer. Hence, it needs to be there in parallel with the root reducer. Is that right?

Reply:
Yup exactly

Reply:
Okay, in that case, we can leave it as is for now.